releasing/build.sh

```diff
@@ -18,5 +18,5 @@ set -e
 
 cd $(dirname "$0")/..
 
-./.github/bin/github-pages-setup.sh
-./.github/bin/github-releases-setup.sh /out
+source ./.github/bin/github-releases-setup.sh /out
```

Comment:
sourcing this to get access to the `PREFIX_BIN` variable looks very hacky to me.
Do you see another way ?

Reply:
This felt like a hack to me as well. I moved the invocation of `github-pages-setup` to `github-releases-setup` so that it is invoked after running the `install` target but before generating the documentation.